frontend/src/router/index.ts

```diff
@@ -1,12 +1,28 @@
 import { createRouter, createWebHistory } from 'vue-router'
-import VideoUploadComponent from '@/components/VideoUploadComponent.vue'
+import ProjectEditor from '@/components/ProjectEditor.vue'
+import ProjectPicker from "@/components/ProjectPicker.vue";
 const router = createRouter({
   history: createWebHistory(import.meta.env.BASE_URL),
   routes: [
     {
       path: '/',
-      name: 'home',
-      component: VideoUploadComponent
+      redirect: '/projects'  // Redirect root path to /projects
+    },
+    {
+      path: '/projects/new',
+      name: 'NewProject',
+      component: ProjectEditor
```

Comment:
Bei größeren SPAs (und ehrlich gesagt benutze ich das bei default) sollte man dynamische Imports verwenden damit der Bundler besser treeshaken uns splitten kann.

Also in dem Fall aus
```js
component: ProjectEditor
```
mit nem Dynamischen Import folgendes machen:
```js
component: () => import('@/components/ProjectEditor.vue')
```

So wird kann dieser Component (falls möglich, sind noch andere Faktoren dran) in nen eigenes Bundle geworfen werden und im besten Fall laden die Seiten schneller weil nurnoch der Code mitgeladen wird, der auch wirklich benutzt wird.
Sonst kann es passieren das Komponenten zusammengeklebt werden, obwohl die garnicht gleichzeitig gebraucht werden.